Receipt/Sources/ListReceiptController.swift

```diff
@@ -39,13 +39,19 @@ public final class ListReceiptController: UITableViewController {
         title = "title_receipts".localized()
         largeTitle()
         setViewBackgroundColor()
-
         navigationItem.backBarButtonItem = UIBarButtonItem.back
+        initializePresenter()
+        guard let presenter = presenter else {
```

Comment:
No need of this guard statement. As there's no mandatory data required to initialize this UI.